src/test/java/org/prebid/server/it/VisxTest.java

```diff
@@ -0,0 +1,54 @@
+package org.prebid.server.it;
+
+import io.restassured.response.Response;
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static io.restassured.RestAssured.given;
+import static java.util.Collections.singletonList;
+
+@RunWith(SpringRunner.class)
+public class VisxTest extends IntegrationTest {
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromVisx() throws IOException, JSONException {
+        // given
+        // VisxTest bid response for imp 001
+        wireMockRule.stubFor(post(urlPathEqualTo("/visx-exchange"))
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/visx/test-visx-bid-request.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/visx/test-visx-bid-response.json"))));
+
+        // pre-bid cache
+        wireMockRule.stubFor(post(urlPathEqualTo("/cache"))
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/visx/test-cache-visx-request.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/visx/test-cache-visx-response.json"))));
+
+        // when
+        final Response response = given(spec)
+                .header("Referer", "http://www.example.com")
+                .header("X-Forwarded-For", "192.168.244.1")
+                .header("User-Agent", "userAgent")
+                .header("Origin", "http://www.example.com")
+                // this uids cookie value stands for {"uids":{"visx":"YM-UID"}}
```

Comment:
When replacing a uuid, please also replace the value of a bidder to either some kind of initials (when the name is too long) or capitalized bidder name (like this one - VISX - it's quite short).

Also, you haven't changed the uids value below  `.cookie("uids", "eyJ1aWRzIjp7InlpZWxkbW8iOiJZTS1VSUQifX0=")` , so if we decode, it will stand for  - `{"uids":{"yieldmo":"YM-UID"}}`, instead of `{"uids":{"visx":"VISX-UID"}}`. 

Please correct this moments - you'll need to encode the text you've changed and put it in cookie value as well as replace `user.buyeruid` in some expected .json test files.

Reply:
I don`t have buyeruid in my test. So I guess, I just change uids to proper one ?

Reply:
I should appear when you fix it in cookie. If not than it's still fine, as long as everything is correct and there are no copy-paste errors.